src/App.tsx

```diff
@@ -3,19 +3,29 @@ import { useState } from 'react';
 import { Footer } from './components/footer';
 import { HeaderComponent } from './components/header/header';
 import { EAppPage } from './constants/link-urls';
+import { ETheme, THEME_CLASSES } from './constants/themes';
 import { AboutPage } from './pages/about-page';
 import { ProductsPage } from './pages/products-page';
 import type { AddSelectedProduct, SelectedProducts } from './types/selected-products';
 
-import './App.css';
+import styles from './App.module.css';
 
 const SELECTED_PRODUCTS_KEY = 'selected-products-key';
+const THEME_KEY = 'theme-key';
 
 const App = () => {
+    const defaultTheme = window.matchMedia('(prefers-color-scheme: dark)').matches ? ETheme.DARK : ETheme.LIGHT;
     const selectedProductsStorage = JSON.parse(localStorage.getItem(SELECTED_PRODUCTS_KEY) || JSON.stringify({}));
+    const selectedTheme = (localStorage.getItem(THEME_KEY) as ETheme | null) || defaultTheme;
 
     const [currentPage, setCurrentPage] = useState<EAppPage>(EAppPage.ABOUT);
     const [selectedProducts, setSelectedProducts] = useState<SelectedProducts>(selectedProductsStorage);
+    const [currentTheme, setCurrentTheme] = useState<ETheme>(selectedTheme);
+
+    const onThemeChange = (theme: ETheme) => {
```

Comment:
create change theme components and mov logic 

in app only get from lockalstorage theme type

Reply:
fixed